runtime/src/accounts.rs

```diff
@@ -778,6 +778,42 @@ impl Accounts {
         )
     }
 
+    fn calc_scan_result_size(account: &AccountSharedData) -> usize {
+        account.data().len()
+            + std::mem::size_of::<AccountSharedData>()
+            + std::mem::size_of::<Pubkey>()
+    }
+
+    /// Accumulate size of (pubkey + account) into sum.
+    /// Return true iff sum > 'byte_limit_for_scan'
+    fn accumulate_and_check_scan_result_size(
```

Comment:
This method name doesn't really speak to me self-documentation-wise.  Let me noodle on it for a bit

Reply:
trent says go ahead